pyfr/readers/base.py

```diff
@@ -22,3 +26,195 @@ def to_pyfrm(self):
         mesh['mesh_uuid'] = np.array(str(uuid.uuid4()), dtype='S')
 
         return mesh
+
+
+class MeshDBAssembler(object):
```

Comment:
Good question.  NodalMeshAssembler, perhaps, to indicate that it is for meshes specified in terms of a bunch of nodes?